diamond_agent/tests/test_single_node.py

```diff
@@ -196,6 +197,12 @@ def test_uninstall_workflow(self):
             self.fail('diamond process not running')
         self.assertFalse(psutil.pid_exists(pid))
 
+        # Check if uninstall cleans up after diamond
+        for path_name in _PATHS_TO_CLEAN_UP:
```

Comment:
For sanity, let's add a similar check before uninstall is called that asserts these path do exist